### PR TITLE
Dev/landing page

### PR DESCRIPTION
**Pull request by mastercodingbear**
Not finished unit test.

### CODE REVIEW

**Comment by Matt-Mckeller** [comment]
the #1 most important thing is that the page is not responsive right now

I'll work on getting a designer on this asap. The designer who created this is working on the pricing page and has a full time job m-f so he can only work weekends

Maybe we can still get this to look a little better on smaller screens before we get a complete mobile design?

**Comment by Matt-Mckeller** [comment]
not a fan of the animation on the hero section, seems distracting

maybe if it was slowed down like 10-20x but I would have to see that

**Comment by Matt-Mckeller** [comment]
i like the quote movement on the testemonial

**Comment by Matt-Mckeller** [comment]
could I see some other options for the explore / get started button animation?

**Comment by Matt-Mckeller** [comment]
updated the button text on get started

**Review by Matt-Mckeller — COMMENTED** [review]
I kind of like the ripple / wave effect below the our services section but I think the green background does not look nearly as good with this. I think the positioning of the green blob in the background and the section divider is really important to the visual appeal

Maybe something could be done to improve it but I'm not sure. Would have to see some examples

----

I think the animations for the process section and statistics section are kinda cool, but could we take them off a timer and instead make them work with the scroll position? as user scrolls down animation progresses, scroll up maybe it reverts ( or maybe it stays finished ) 

the timing for the initiation of the animations for the process and statistics section seem off too. I actually am having trouble viewing them because they start before my scrolling can get there 

I know the animations are extras!!! I really really appreciate the work on those. I think the interactivity/motion added to this page really takes it up another level

----
For the buttons I think that if the arrow also changed color it would look much better

button arrow and text alignment is off ( and was also off on mockup, I did some updates for that too )

--
i love the quote movement on hover 


Fantastic work! I'm so happy to see this come to life!!! You're doing great and I could not be happier :)

I would say that we can put the unit testing off until the rest is at 100%. I definitely want unit testing to be part of our standard process, and to be included in Expanse's website source code but most important is going to be getting this functional and usable. My vision is to make this an evolving code base which we reference as a template / example for future projects. This will take some time to get right XD

File: components/ui/buttons/call-to-action-button.tsx [hunk 1]
```diff
@@ -1,6 +1,57 @@
-type Props = {
-    text: string;
+import styled from 'styled-components';
+import Arrow from '../../../public/assets/icons/Arrow.svg';
+
+export type Props= {
+  text: string
 }
+
+const CallButton = styled.button`
```
Comment (Matt-Mckeller) [inline components/ui/buttons/call-to-action-button.tsx:8]: StyledCallToActionButton ?

File: components/ui/containers/component-containers/home/consultation.component.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            display: "flex",
+            flexDirection: "column",
+            justifyContent: "flex-start",
+            padding: 30,
+            width: "33%"
+        },
+        title: {
+            fontSize: 28,
+            textTransform: "uppercase",
+            color: "Silver"
```
Comment (Matt-Mckeller) [inline components/ui/containers/component-containers/home/consultation.component.tsx:19]: can we put this into the color config file and reference it there? also would prefer the hex version #C6C6C6

File: components/ui/containers/component-containers/home/consultation.component.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    margin-bottom: 30px;
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+`;
+
+const StepHeader = styled.h1`
+    font-size: 96px;
+    color: #DBD1FE;
```
Comment (Matt-Mckeller) [inline components/ui/containers/component-containers/home/consultation.component.tsx:37]: color config or some centralized location pls

File: components/ui/containers/page-sections/home/testimonial-section.tsx [hunk 4]
```diff
@@ -1,9 +1,207 @@
...
+import { useEffect } from 'react';
+import styled from "styled-components";
+import { createStyles, makeStyles, Theme } from "@material-ui/core";
+import Fab from '@material-ui/core/Fab';
+import ArrowForwardIosRoundedIcon from '@material-ui/icons/ArrowForwardIosRounded';
+import ArrowBackIosRoundedIcon from '@material-ui/icons/ArrowBackIosRounded';
+import Carousel from 'react-multi-carousel';
+import { ButtonGroupProps, DotProps } from 'react-multi-carousel/lib/types';
+import 'react-multi-carousel/lib/styles.css';
+// import clsx from  'clsx';
```
Comment (Matt-Mckeller) [inline components/ui/containers/page-sections/home/testimonial-section.tsx:11]: remove?

File: components/ui/layout/header.tsx [hunk 5]
```diff
@@ -61,8 +61,8 @@ const Header = () => {
   const isCurrentPath = (path: string) => {
     return path === router.pathname;
   }
-  console.log({router});
-  console.log(router.pathname, router.basePath);
+  // console.log({router});
+  // console.log(router.pathname, router.basePath);
```
Comment (Matt-Mckeller) [inline components/ui/layout/header.tsx:65]: remove when finished

**Comment by Matt-Mckeller** [comment]
gsap-bonus.tgz needed?

File: next.config.js [hunk 6]
```diff
@@ -1,5 +1,5 @@
-
 // next.config.js
 // const withImages = require('next-images')
```
Comment (Matt-Mckeller) [inline next.config.js:2]: remove

File: package.json [hunk 7]
```diff
@@ -6,33 +6,46 @@
...
   },
   "dependencies": {
     "@material-ui/core": "^4.11.4",
+    "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.58",
+    "aos": "^2.3.4",
     "classnames": "2.2.6",
     "date-fns": "2.10.0",
     "gray-matter": "4.0.2",
+    "gsap": "./gsap-bonus.tgz",
```
Comment (Matt-Mckeller) [inline package.json:20]: any reason we are importing it like this? can we reference a repo?

File: pages/_document.tsx [hunk 8]
```diff
@@ -1,47 +1,68 @@
...
         </body>
       </Html>
-    )
+    );
   }
 }
+
+// `getInitialProps` belongs to `_document` (instead of `_app`),
+// it's compatible with server-side generation (SSG).
+MyDocument.getInitialProps = async (ctx) => {
```
Comment (Matt-Mckeller) [inline pages/_document.tsx:29]: fantastic, good comments

File: pages/index.tsx [hunk 9]
```diff
@@ -46,36 +85,47 @@ export const Index = ({
...
-      <SectionContainerStyled backgroundColor="teal">
+      <SectionContainerStyled height="inherit" backgroundColor={BACKGROUND_SECONDARY}>
         <TestimonialSection {...testimonialProps} />
       </SectionContainerStyled>
       <GlobalStyle/>
     </BaseFlexContainer>
   )
 }
 
+// Index.getInitialProps = ({ req }) => {
```
Comment (Matt-Mckeller) [inline pages/index.tsx:116]: ?  Don't we need this for server side rendering?

File: pages/_app.tsx [hunk 10]
```diff
@@ -1,16 +1,34 @@
...
+        });
+      }, []);
     return (
-        <>
-            <Meta/>
+    <>
+        <Meta/>
+        <ThemeProvider theme={theme}>
+            {/* CssBaseline kickstart an elegant, consistent, and simple baseline to build upon. */}
+            <CssBaseline />
```
Comment (Matt-Mckeller) [inline pages/_app.tsx:27]: Oh! Cool! Did you also remove the normalize css?

File: components/ui/layout/header.tsx (hunk 5 above)
Comment (Matt-Mckeller) [reply components/ui/layout/header.tsx:65]: oh, these are my logs lol

**Review by Matt-Mckeller — CHANGES_REQUESTED** [review]
:) see comments! Great job, I can tell you put a lot of work into this and I appreciate it

**Comment by Matt-Mckeller** [comment]
also please make sure this works on all modern browsers, I did not test this

**Review by Matt-Mckeller — APPROVED** [review]

File: package.json (hunk 7 above)
Comment (mastercodingbear) [reply package.json:21]: drawing line animation and counter animation are implemented by gsap

**Comment by mastercodingbear** [comment]
> gsap-bonus.tgz needed?

this has bonus plugins.
like ScrollTrigger, Textplugin. And I thinkg we can make more animations with gsap.

**Comment by Matt-Mckeller** [comment]
> > gsap-bonus.tgz needed?
> 
> this has bonus plugins.
> like ScrollTrigger, Textplugin. And I thinkg we can make more animations with gsap.

can we import from github instead of a tgz file?

**Pull request merged**